### ISSUE TITLE
Black & White Movies

### ISSUE THREAD

**Issue by kizzjay1** [MetalChris/repository.metalchris#7]
Hi Chris! sorry to bother you on here again but im not sure of any other way to talk to you. i wanted to let you know that black & white movies version 0.0.2 will not download. it keeps saying installation failed. i was able to push version 0.0.1 to my fire stick but that isnt updating either. version 0.0.1 is working fine for me so im just gonna keep it on there & im sure it will update in the future. i just wanted to let you know what is going on. going forward, if i see something, is it okay to communicate that to you through here or would you prefer a different method?

***EDIT: i also cannot log into carbontv anymore. i was able to yesterday though. ive tried 4 different times making sure the login info is entered correctly but it will not let me into the addon.

**Comment by MetalChris** [comment]
No bother.  If something breaks, or if I screw something up I need to know.  In the case of B&W Movies, I screwed up.  Version 0.0.2 is not ready yet, but I accidentally upped the version number when updating the repo for a different add-on.  I've since updated the repo again, which should now reflect the correct version (0.0.1) for B&W Movies.

I was able to log in to the CarbonTV add-on using a couple of my systems without issue.  Maybe they were doing some site maintenance that caused some problems for you.
